### PR TITLE
Listing supported versions on failed rosa creates

### PR DESCRIPTION
**Pull request by bradmwilliams**

### CODE REVIEW

**Review by AlexNPavel — APPROVED** [review]

File: pkg/manager/manager.go [hunk 1]
```diff
@@ -1950,6 +1950,15 @@ func (m *jobManager) lookupRosaVersions(prefix string) []string {
 	return matchedVersions
 }
 
+func (m *jobManager) getSupportedRosaVersions() string {
+	m.rosaVersions.lock.RLock()
+	defer m.rosaVersions.lock.RUnlock()
+	var versions []string
+	versions = append(versions, m.rosaVersions.versions...)
+	sort.Sort(sort.Reverse(sort.StringSlice(versions)))
+	return strings.Join(versions, ",")
```
Comment (bparees) [inline pkg/manager/manager.go:1959]:
```suggestion
	return strings.Join(versions, ", ")
```

Comment (bradmwilliams) [reply pkg/manager/manager.go]: Done!

**Comment by bparees** [comment]
/lgtm

@AlexNPavel was your hold because of my comment or something else?

**Pull request merged**